site/content/3.12/concepts/data-structure/documents/computed-values.md

```diff
@@ -74,12 +70,174 @@ Each object represents a computed value and can have the following attributes:
+~addIgnoreCollection(coll.name());
+```
+To remove the computed values configuration from a collection, set the
+`computedValues` property to `null` or `[]` (empty array):
+
+```js
+---
+name: arangosh_unset_collection_properties_computed_values
+description: ''
+---
+~var coll = db._collection("compValCollection");
+/* Remove the computed values of an existing collection */
+db.compValCollection.properties({ computedValues: null });
+~removeIgnoreCollection(coll.name());
+~db._drop(coll.name());
+```
+
+See [`db._create()`](../../../develop/javascript-api/@arangodb/db-object.md#db_createcollection-name--properties--type--options)
+and [`collection.properties()`](../../../develop/javascript-api/@arangodb/collection-object.md#collectionpropertiesproperties)
+in the _JavaScript API_ for details.
+{{< /tab >}}
+
+{{< tab "cURL" >}}
+```sh
+curl -XPUT -d '{"computedValues":[{"name":"title","expression":"RETURN \"TBA\"","overwrite":false,"computeOn":["insert","update","replace"],"failOnWarning":false,"keepNull":true}]' http://localhost:8529/_db/mydb/_api/collection/coll/properties
+```
+
+See the [`PUT /_db/{database-name}/_api/collection/{collection-name}/properties`](../../../develop/http-api/collections.md#change-the-properties-of-a-collection)
+endpoint in the _HTTP API_for details.
+{{< /tab >}}
+
+{{< tab "JavaScript" >}}
+```js
+let coll = db.collection("coll");
+const props = await coll.properties({
+  computedValues: [
+    {
+      name: "title",
+      expression: "RETURN \"TBA\"",
+      overwrite: false,
+      computeOn: ["insert", "update", "replace"],
+      failOnWarning: false,
+      keepNull: true
+    }
+  ]
+});
+```
+
+See [`DocumentCollection.properties()`](https://arangodb.github.io/arangojs/latest/interfaces/collection.DocumentCollection.html#properties.properties-2)
+in the _arangojs_ documentation for details.
+{{< /tab >}}
+
+{{< tab "Go" >}}
+```go
+ctx := context.Background()
+coll, err := db.GetCollection(ctx, "coll", nil)
+err = coll.SetProperties(ctx, arangodb.SetCollectionPropertiesOptions{
+  ComputedValues: []arangodb.ComputedValue {
+    {
+      Name: "title",
+      Expression: "RETURN \"TBA\"",
+      Overwrite: false,
+      ComputeOn: []arangodb.ComputeOn {
+        arangodb.ComputeOnInsert,
+        arangodb.ComputeOnUpdate,
+        arangodb.ComputeOnReplace,
+      },
+      FailOnWarning: utils.NewType(false), // pointer to bool
+      KeepNull: utils.NewType(true), // pointer to bool
+    },
+  },
+})
+```
+
+See [`Collection.SetProperties()`](https://pkg.go.dev/github.com/arangodb/go-driver/v2/arangodb#Collection)
+in the _go-driver_ v2 documentation for details.
+{{< /tab >}}
+
+{{< tab "Java" >}}
+```java
+CollectionPropertiesOptions props = new CollectionPropertiesOptions()
+  .computedValues(new ComputedValue()
+    .name("title")
+    .expression("RETURN \"TBA\"")
+    .overwrite(false)
+    .computeOn(ComputeOn.insert, ComputeOn.update, ComputeOn.replace)
```

Comment:
@rashtao I added the examples for Computed Values now too.
Is it necessary to use `ComputedValue.ComputeOn.*` here to refer to the elements of the enum?

Reply:
```suggestion
    .computeOn(ComputedValue.ComputeOn.insert, ComputedValue.ComputeOn.update, ComputedValue.ComputeOn.replace)
```